package.json

```diff
@@ -63,6 +65,7 @@
     "client-oauth2": "^4.3.3",
     "csrf-csrf": "^2.2.2",
     "dotenv": "^16.0.3",
+    "encoding": "^0.1.13",
```

Comment:
Not sure where this is used, unless it's some side effect of the previous GCP dependencies above and this is bumping their version numbers.